README.md

```diff
@@ -30,11 +30,11 @@ Optionally, assert a particular value as well. The return value is available for
     expect($('body')).to.have.prop('value').match(/bar/);
 
 ### `css(name[, value])`
-Assert that the first element of the selection has the given CSS property, using [`.css()`](http://api.jquery.com/css/).
+Assert that the first element of the selection has the given CSS property, using [`.css()`](http://api.jquery.com/css/). Note that jQuery returns computed (rgb) values for colors, and evaluating against a hex or string will return a test failure.
```

Comment:
The linked page, [http://api.jquery.com/css/](http://api.jquery.com/css/) contains this information. I don't think we need to document it here as well. It would be good to mention computed style or value though so I suggest updating the line below to:
```
Optionally, assert the computed value as well.
```

I do still agree with changing the example though as that's a more likely return value